datasets/bomen/acc/gebrekregistratie/v1.0.0.json

```diff
@@ -26,7 +26,7 @@
         "type": "integer",
         "description": "Unieke aanduiding van record"
       },
-      "boomVeiligheidsinspectieId": {
+      "boomveiligheidsinspectieId": {
```

Comment:
De correcte manier voor deze verandering is om een nieuwe JSON file v1.0.1.json aan te maken met de verandering.